src/register/RegistrationPage.jsx

```diff
@@ -502,13 +502,6 @@ class RegistrationPage extends React.Component {
           value: this.state.totalRegistrationTime,
         },
       });
-
-      if (window.optimizelyExperimentName !== 'release-pp-concept-1') {
-        window.optimizely.push({
-          type: 'event',
-          eventName: 'authn_pp_conversion',
```

Comment:
Do we need to remove this event from welcome page on submit/skip click as well?

Reply:
yeah will remove it